src/main/templates/ratings.hbs

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  {{>header}}
+  <!-- Main content starts - do not edit above this line -->
+    <table id="sorttable" class="table table-small table-hover display">
+      <thead class="thead-light">
+        <tr>
+          <th scope="col" class="align-text-top">Region</th>
+          <th scope="col" class="align-text-top" width="10%">DCNC Code <small>(For CTT)</small></th>
+          <th scope="col" class="align-text-top" width="15%">Rating Label Value <small>(For DCP Metadata)</small></th>
+          <th scope="col" class="align-text-top">Rating Agency Value <small>(For DCP Metadata)</small></th>
+          <th scope="col" class="align-text-top">Rating Agency</th>
+          <th scope="col" class="align-text-top">Reference</th>
```

Comment:
Suggest narrowing the column

Reply:
This is for column width only on full screen to preserve the column width not wrapping unnecessarily, other screen sizes are unaffected by this.

Reply:
Removed from added `<br>` in header in 1e9af3f